test/services/postcard.spec.js

```diff
@@ -2,6 +2,7 @@
 
 var expect  = require('chai').expect;
 var Angular = require('angular');
+var Config  = require('../../config');
```

Comment:
ah dang, i shouldve caught this in the other pr 😬 same as the other comment, you should move this after the module requires.